server/methods/sendForgotPasswordEmail.js

```diff
@@ -5,6 +5,9 @@ Meteor.methods({
 		email = email.trim();
 
 		const user = RocketChat.models.Users.findOneByEmailAddress(email);
+		if (!user) {
```

Comment:
I think you can use an `else` in the `if` below, it will check for `user` once one time.